crypto/libsodium-fork/test/default/sign.c

```diff
@@ -1057,6 +1057,337 @@ static void add_l(unsigned char * const S)
     sodium_add(S, l, sizeof l);
 }
 
+
+
+// Test vector according to : "Taming the many EdDSAs" white paper
```

Comment:
in case you're looking for more test vectors, I found some in https://gist.github.com/hdevalence/93ed42d17ecab8e42138b213812c8cc7